include/modem/lte_lc.h

```diff
@@ -347,40 +370,32 @@ struct lte_lc_cell {
 	int16_t rsrq;
 };
 
-struct lte_lc_ncell {
-	/** EARFCN of the neighbour cell, per 3GPP TS 36.101. */
-	uint32_t earfcn;
 
-	/** Difference in milliseconds of serving cell and neighbor cell
-	 *  measurement, in the range -99999 ms < time_diff < 99999 ms.
+/** @brief Structure containing results of neighbor cell measurements. */
+struct lte_lc_cells_info {
+	/** The current cell information is valid if the current cell ID is not
+	 *  set to LTE_LC_CELL_EUTRAN_ID_INVALID.
 	 */
-	int time_diff;
-
-	/** Physical cell ID. */
-	uint16_t phys_cell_id;
+	struct lte_lc_cell current_cell;
 
-	/** RSRP of the neighbor cell. Format is the same as for RSRP member
-	 *  of struct lte_lc_cell.
+	/** The ncells_count member indicates whether or not the neighbor_cells structure
+	 *  contains valid neighbor cell information. If it is zero, no neighbor cells were found
+	 *  for the current cell.
 	 */
-	int16_t rsrp;
+	uint8_t ncells_count;
 
-	/** RSRQ of the neighbor cell. Format is the same as for RSRQ member
-	 *  of struct lte_lc_cell.
+	/** Neighbor cells for the current cell. */
+	struct lte_lc_ncell *neighbor_cells;
+
+	/** The gci_cells_count member indicates whether or not the gci_cells structure contains
```

Comment:
> @peknis Could you comment on how to best write the names of structs in doxygen comments? I've been using `` ``member_name`` ``, but don't know if that's the way to go

I'll get back to this asap.

Reply:
Okay, here comes:
- for structs - it depends if you want to link to them, then use @ref <struct>
- when linking to members, what should work is @ref <struct>::<member>
- there might be cases when you don't want to create a link because you are mentioning the struct in its own documentation, so linking would not make much sense
- then it's best to use @c <struct> , although it will produce the same output as backticks so backticks are fine as well